### ISSUE TITLE
Revising the Mock-ups

### ISSUE THREAD

**Issue by CahidArda** [bounswe/bounswe2022group7#247]
As part of the first week of CMPE451, we will revise the mockups prepared in CMPE352.

We have done some updates on the requirements. This means that the indexes of the mock-up acceptance criteria shall be updated. Additionally, we realised that we have overcomplicated some aspects of the app in our mockups.

We are planning to do the following changes:
- Update images:
    - Remove map & location related features
    - Remove payment & fee related features
    - Improve mobile mock-ups
- Update acceptance criteria

**Status of Scenarios and Mockups:**
- [x] [Scenario 1: Creating Events](https://github.com/bounswe/bounswe2022group7/wiki/Scenario-1:-Creating-Events)
- [x] [Scenario 2](https://github.com/bounswe/bounswe2022group7/wiki/Scenario-2)
- [x] [Scenario 3: Copyright Infringement Report](https://github.com/bounswe/bounswe2022group7/wiki/Scenario-3:-Copyright-Infringement-Report)

Deadline: 11.10.2022, 13.00
Reviewer: @askabderon

**Comment by erimerkin** [comment]
I have [updated Mockups and scenario details for Scenario 1](https://github.com/bounswe/bounswe2022group7/wiki/Scenario-1:-Creating-Events/f77882738af7928ca663f7b6e21bd148dc6fa2bd). This update included removal of acceptance criterias that is no longer in requirements, removal of map and location specific parts of mockup.

**Comment by CahidArda** [comment]
For scenario 2, we updated the requirements by removing the sub items of 1.2.4.3 since these sub items were removed in requirements revision. We also updated the images to reflect the changes we made in the requirements: removed maps, distance to event information sections and event fees,

For scenario 3, we updated the UI by adding a new button to the bottom navigation bar. This button will allow the user to add content to the website.
